ecosystem_cicd_tools/validations.py

```diff
@@ -226,6 +223,12 @@ def get_version_py(plugin_directory):
                 'Failed to get version from file __version__.py')
 
 
+def is_valid_plugin_package_name(f):
+    return (f == 'fabric_plugin') or (f.startswith('cloudify_') and 
+                                      'plugin' not in f and 
+                                      'sdk' not in f) 
```

Comment:
What about `serverless_plugin`?
https://github.com/cloudify-incubator/cloudify-serverless-plugin
I think we can remove line 228.